packages/marketplace/src/styles/pages/apps.scss

```diff
@@ -0,0 +1,4 @@
+.app-list {
+  height: 100vh;
+  overflow: auto;
```

Comment:
Two things, vh doesn't work in IE - can you check the page isn't broken in IE. Also, is overflow not auto by default?

Reply:
Use `overflow: auto` to override the overflow value from parent element (current is `overflow: visible` ).
I have tested on IE (local env) and it work.